coalib/parsing/InvalidFilterException.py

```diff
@@ -1,2 +1,9 @@
 class InvalidFilterException(Exception):
-    pass
+    def __init__(self, filter, available_filters):
+        self.filter = filter
+        self.available_filters = available_filters
```

Comment:
can't you just call `super().__init__(message)`? Then you don't need to override `__str__`.